flexget/plugins/output/slack.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.utils import json
+from flexget.utils.template import RenderError
+
+log = logging.getLogger('slack')
+
+
+class OutputSlack(object):
+    """
+    Example:
+
+      slack:
+        webhook-url: <string>
+        [text: <string>] (default: "{{task}} - Download started: {{series_name}} {{series_id}}" -- accepts Jinja2)
```

Comment:
This will crash if those value are not present. Better to remove them or use some if condition.